### PR TITLE
separate upload into new cli, write dsa annots to disk

### PR DESCRIPTION
**Pull request by druvpatel**
- separated dsa annotation upload and building into two diff CLIs
- made CLIs lightweight to allow for function calling through dask/work scheduler
- added back writes to file and "output_folder" entry in data_config

### CODE REVIEW

File: data_processing/pathology/cli/dsa/dsa_api_handler.py [hunk 1]
```diff
@@ -6,28 +6,50 @@
...
+    return None
+    
+
+def get_item_uuid(image_name, uri, token, collection_name):
     """
     Returns the DSA item uuid from the provided image_name
     Expects image_name to be the image name with svs extension.
     """
+
+    collection_uuid = get_collection_uuid(uri, token, collection_name)
```
Comment (doori) [inline data_processing/pathology/cli/dsa/dsa_api_handler.py:34]: if `collection_uuid` is None, do we want to return None here?

File: data_processing/pathology/cli/dsa/dsa_viz.py [hunk 2]
```diff
@@ -39,49 +41,49 @@ def save_push_results(base_annotation, elements, annotation_name, image_filename
...
+                "stardist-cell": stardist_cell, 
+                "regional-polygon": regional_polygon, 
+                "qupath-polygon": qupath_polygon, 
+                "bitmask-polygon": bitmask_polygon,
+                "heatmap": heatmap}
+    
+    if source_type not in functions:
+            print(f"ERROR: source_type {source_type} is not supported")
+    else:   
+        annotation_filepath = functions[source_type](data_config)
```
Comment (doori) [inline data_processing/pathology/cli/dsa/dsa_viz.py:82]: Nice

**Review by doori — APPROVED** [review]
Looks great!

File: data_processing/pathology/cli/dsa/dsa_viz.py (hunk 2 above)
Comment (druvpatel) [reply data_processing/pathology/cli/dsa/dsa_viz.py:82]: Haha looked up some pythonic alternatives to if/else..

**Pull request merged**